File: python/examples/post_services_info.py
import pprint
import os
import sys
import json

sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))

from request import BaseRequest
from dotenv import dotenv_values

config = dotenv_values()

# Example of POST /services/:id/info

class PostServicesInfoRequest(BaseRequest):
    def __init__(self):
        super().__init__()
    
    def post_info(self, service_id, service_info):
        return self.make_request('POST', f'/services/{service_id}/info', body=service_info)

service_id = config['SERVICE_ID'];
cwd = os.getcwd();
file_path = f"{cwd}/../data/services/add_info_corp.json";

with open(file_path, 'r') as file:
    service_info = json.load(file)
    request = PostServicesInfoRequest()
    response = request.post_info(service_id, service_info)
    pprint.pprint(response)

File: php/examples/post_services_info.php
<?php
require_once 'base_request.php';

// Example of POST /services/:service_id/info

$debug = true;
$service_id = $_ENV['SERVICE_ID'];

$file_path = '../data/services/add_info_corp.json';
$service_info = file_get_contents($file_path);
$request_body = json_decode($service_info);
$request_params = '';

$path = '/services/' . $service_id . '/info';

send_request('POST', $path, $request_params, json_encode($request_body));